woocommerce/woocommerce
# Fix accessibility name for individual product checkboxes in the grouped product page

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Closes #51928



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Add 2/3 products, mark the type as "Single Product" and make sure to check "Limit purchases to 1 item per order" found in the `Inventory` tab
2. Create a new product, mark the type as "Grouped Product" and add the products created in Step 1 from the `Linked Products` tab.
3. Visit the product page, checkboxes should show for each linked product:

<img width="1211" alt="Screenshot 2024-10-08 at 16 30 38" src="https://github.com/user-attachments/assets/2e5e1700-3655-45ee-a9bd-ebe00602782d">

4. Inspect the checkbox and look for a label with the "screen-reader-text" class name:

```
<label for="quantity-12" class="screen-reader-text">Buy one of Yellow Can for €19</label>
```



### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 

Fix accessibility name for individual product checkboxes in the grouped product page

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
